core/store/tx_manager.go

```diff
@@ -753,31 +753,32 @@ func (txm *EthTxManager) BumpGasByIncrement(originalGasPrice *big.Int) *big.Int
 	return BumpGas(txm.config, originalGasPrice)
 }
 
-// BumpGas returns a new gas price increased by the largest of:
-// - A configured percentage bump (ETH_GAS_BUMP_PERCENT)
-// - A configured fixed amount of Wei (ETH_GAS_PRICE_WEI)
-// - The configured default base gas price (ETH_GAS_PRICE_DEFAULT)
+// BumpGas computes the next gas price to attempt as the largest of:
+// - A configured percentage bump (ETH_GAS_BUMP_PERCENT) on top of the baseline price.
+// - A configured fixed amount of Wei (ETH_GAS_PRICE_WEI) on top of the baseline price.
+// The baseline price is the maximum of the previous gas price attempt and the node's current gas price.
 func BumpGas(config orm.ConfigReader, originalGasPrice *big.Int) *big.Int {
```

Comment:
I'm scared that this doesn't have a test case.

Reply:
I'll definitely add one!